bot/core/events/registry.py

```
@@ -0,0 +1,468 @@
+    HIGHEST = 0
+    HIGH = 100
+    NORMAL = 200
+    LOW = 300
+    LOWEST = 400
+
+
+T = TypeVar('T', bound=EventType)
+
+
+@dataclass
+class Event(Generic[T]):
```

Ruff (0.8.2)

Redefinition of unused `Event` from line 9

(F811)